### ISSUE TITLE
When I submit synonyns should I do some kind of "reindexing" ?

### ISSUE THREAD

**Issue by EasyLOB** [typesense/typesense#907]
## Description

I created a collection and **AFTER** loading data I created some Synonyns.
Is some kind of "reindexing" necessary ?

## Metadata

0.24.0
Ubuntu

**Comment by kishorenc** [comment]
No, synonyms are used only during query time, so no need to reindex your data after adding synonyms.
